users/vieting/jobs/fairseq.py

```diff
@@ -2,7 +2,7 @@
 import glob
 import os
 import random
-import subprocess
+import subprocess as sp
```

Comment:
Is subprocess not used anywhere? If it is, these usages would need to be updated if you change the name here.